db/migrate/20190606115037_add_organization_id_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrganizationIdToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :users, :organization
```

Comment:
are u aware that removing the organization_id column will delete all data?

Reply:
So should i change smth?

Reply:
depends if u want to keep existing data or not